package.json

```diff
@@ -43,6 +43,7 @@
     "babel-preset-stage-1": "6.24.1",
     "cypress": "3.1.5",
     "elliptic": "^6.5.0",
+    "eosjs-ecc": "^4.0.7",
```

Comment:
?

Reply:
This is in the dev dependencies for testing compatibility with previous methodologies

Reply:
eosjs-ecc is going away; we shouldn't depend on it.

Reply:
It's not in the produced library; it's only here in devDependencies to verify that what we're producing now matches what it used to produce.